composer.json

```diff
@@ -1,94 +1,93 @@
 {
-  "name": "acquia/drupal-recommended-project",
```

Comment:
These changes aren't functional, it's just a style cleanup to pass ORCA static analysis.